index.html

```diff
@@ -0,0 +1,173 @@
+        <div id="video-container">
+          <iframe
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope;
+          picture-in-picture"
+            allowfullscreen
+          >
+          </iframe>
+          <div>
+            <div id="title-container">
+              <h2>Contributing to Open Source Part I: The Easy Way</h2>
+              <div class="d-flex">
+                <p id="views">
+                  57 231 vues<time datetime="2014-02-24"
+                    >• 27 February, 2014</time
+                  >
+                </p>
+                <div class="d-flex">
+                  <div class="video-icons-container">
+                    <img class="video-icons" src="images/like.svg" alt="like" />
+                    <p>422</p>
+                  </div>
+                  <div class="video-icons-container">
+                    <img
+                      class="video-icons"
+                      src="images/dislike.svg"
+                      alt="dislike"
+                    />
+                    <p>18</p>
+                  </div>
+                  <button id="share" class="video-icons-container">
+                    <img
+                      class="video-icons"
+                      src="images/share.svg"
+                      alt="share"
+                    />
+                    share
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div id="video-description">
+              <div class="d-flex">
+                <div class="video-avatar d-flex">
+                  <p id="profile-icon">T</p>
+                  <div>
+                    <p id="title-sub">The Odin Project</p>
+                    <p id="sub_nom">988 subscribers</p>
+                  </div>
+                </div>
+                <button id="subscribe-btn" type="button">
+                  subscribe
+                </button>
+              </div>
+
+              <p id="tags-videos">
+                How do you contribute to open source? In this video, I'll
+                describe the easiest way to make a simple contribution to an
+                open source project -- without using the command line, a text
+                editor, "forking a repo", or even touching Git outside of
+                Github. I'll also describe what kinds of contributions open
+              </p>
+              <a id="show">Show More</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <aside>
+        <div id="aside-title">
+          <p>Up next</p>
+          <p class="icons d-flex">
+            Autoplay<img class="icons" src="images/switch.svg" alt="switch" />
+          </p>
+        </div>
+        <div class="thumbnail-container">
+          <div id="first-video" class="thumbnail d-flex">
+            <img src="images/1.png" alt="thumbnail_1" />
```

Comment:
I see that the name of your file in the image folder is 1.PNG, that could be the issue why the images are not showing